build-pipeline/aspnet-docker-linux-images.json

```diff
@@ -201,7 +201,7 @@
     "type": "GitHub",
     "name": "aspnet/aspnet-docker",
     "url": "https://github.com/aspnet/aspnet-docker.git",
-    "defaultBranch": "dev",
```

Comment:
You may consider setting these to master everywhere.  Since they are invoked from pipebuild, the branch always is passed when queued.  Having them be the same everywhere may help simplify merging changes.

Reply:
Yeah, I'll do that. I didn't know that pipebuild would override this anyways.